README.md

```diff
@@ -512,3 +525,61 @@ new TextOf(
     new IOException("It doesn't work at all")
 );
 ```
+
+## LinQ Analogy
+### Standard Query Operators
+
+LinQ                  | Yaapii.Atoms
+----------------------|-------------
+**Aggregate**         |*Not aviable yet*
+**All**               | And&lt;T&gt; <!--pre>var allResult = new And&lt;string&gt;(<br>&nbsp;&nbsp;new EnumerableOf&lt;string&gt;() { "A", "B", "C" },<br>&nbsp;&nbsp;(input) => input != "B"<br>); //newFiltered contains A & C</pre-->
+**Any**		       |Or&lt;T&gt;<!--pre>var b = new Contains&lt;string&gt;(<br>&nbsp;&nbsp;new EnumerableOf&lt;string&gt;("Hello", "my", "cat", "is", "missing"),<br>&nbsp;&nbsp;(str) => str == "cat"<br>).Value()); //b = true </pre--> 
+**AsEnumerable**      |<pre>var list = new EnumerableOf&lt;int&gt;(<br>&nbsp;&nbsp;new List&lt;int&gt; { 1, 2, 3, 4 }.GetEnumerator()<br>);</pre>
+**Average**           |<pre>var avg = new AvgOf(1, 2, 3, 4).AsFloat(); //avg = 2.5</pre>
+**Cast**              |*Not aviable yet*
+**Concat**            |<pre>var joined = new Joined&lt;string&gt;(<br>&nbsp;&nbsp;new EnumerableOf&ltstring&gt;("dies", "ist"),<br>&nbsp;&nbsp;new EnumerableOf&ltstring&gt;("ein", "Test")<br>).Value(); //joined = {"dies", "ist", "ein", "Test"}</pre>
```

Comment:
the &lt is missing the trailing ;

please have a look at the document and fix other occurences of this problem too